bin/tester.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+
+const colors = require('colors');
+const test = require('../lib/test');
```

Comment:
If we move the `lib/test` to `src` then this import will be `../dist/test`.

We can use JS for `bin/tester` because we'll be writing most of the code inside `src`.

Reply:
Now the global `tester` command is linked to a script in the `dist` folder, I'm not sure if that's a good idea or not. But I like keeping the source in TypeScript.

Reply:
How about putting commands in `src/bin` ?

Reply:
Sorry my bad, I forgot the dist folder is packaged by npm, all good